src/custom/pages/Claim/index.tsx

```diff
@@ -204,19 +204,25 @@ export default function Claim() {
     OperationType.APPROVE_TOKEN
   )
 
-  const [gnoApproveState, gnoApproveCallback] = useApproveCallbackFromClaim(
-    () => openModal(GNO_CLAIM_APPROVE_MESSAGE, OperationType.APPROVE_TOKEN),
-    closeModal,
+  const [gnoApproveState, gnoApproveCallback] = useApproveCallbackFromClaim({
+    openTransactionConfirmationModal: () => openModal(GNO_CLAIM_APPROVE_MESSAGE, OperationType.APPROVE_TOKEN),
+    closeModals: closeModal,
     // approve max unit256 amount
-    isSupportedChain(chainId) ? CurrencyAmount.fromRawAmount(GNO[chainId], MaxUint256) : undefined
-  )
-
-  const [usdcApproveState, usdcApproveCallback] = useApproveCallbackFromClaim(
-    () => openModal(USDC_CLAIM_APPROVE_MESSAGE, OperationType.APPROVE_TOKEN),
-    closeModal,
+    amountToApprove: isSupportedChain(chainId) ? CurrencyAmount.fromRawAmount(GNO[chainId], MaxUint256) : undefined,
+    // TODO: enable, fix this
+    // amountToCheckAgainstAllowance: investmentAmountAsCurrency,
```

Comment:
okay, this is the placeholder for next PR 🎉

Reply:
@W3stside u are adding the `amountToCheckAgainstAllowance` in another PR, right?

Reply:
yeah